frontend/src/features/recommendation/SearchFilter.tsx

```diff
@@ -65,6 +66,12 @@ function SearchFilter() {
         Filters
       </Button>
       <Collapse in={expanded} timeout="auto" unmountOnExit>
+        {searchParams.get('uploader') && (
+          <UploaderFilter
+            value={searchParams.get('uploader') ?? ''}
+            onDelete={() => setFilter('uploader', '')}
+          />
```

Comment:
What about adding a little margin-bottom here?
```suggestion
          <Box marginBottom={1}>
            <UploaderFilter
              value={searchParams.get('uploader') ?? ''}
              onDelete={() => setFilter('uploader', '')}
            />
          </Box>
```